src/tools/ReadyToRun.SuperIlc/BuildFolderSet.cs

```diff
@@ -36,16 +38,16 @@ public FrameworkExclusion(string simpleName, string reason)
             new FrameworkExclusion("xunit.performance.api", "Not a framework assembly"),
 
             // TODO (DavidWr): IBC-related failures
-            new FrameworkExclusion("Microsoft.CodeAnalysis.CSharp", "Ibc TypeToken 6200019a has type token which resolves to a nil token"),
-            new FrameworkExclusion("Microsoft.CodeAnalysis", "Ibc TypeToken 620001af unable to find external typedef"),
-            new FrameworkExclusion("Microsoft.CodeAnalysis.VisualBasic", "Ibc TypeToken 620002ce unable to find external typedef"),
+            new FrameworkExclusion("Microsoft.CodeAnalysis.CSharp", "Ibc TypeToken 6200019a has type token which resolves to a nil token", crossgen2Only: true),
```

Comment:
I was thinking for a bit that it would be nice to have a command line option to disable the exclusions so that I don't have to comment them out when testing changes in the crossgen2, but I have came to the conclusion that it is not worth it - commenting out these locally is just fine for my purposes.